src/uct/ib/rdmacm/rdmacm_iface.h

```diff
@@ -34,13 +35,14 @@ struct uct_rdmacm_iface {
     uct_sockaddr_conn_request_callback_t conn_request_cb;
     uint32_t                             cb_flags;
 
-    uct_rdmacm_ep_t                      *ep;
-
     /** Field used only for client side */
     ucs_list_link_t                      pending_eps_list;
+    ucs_list_link_t                      used_cm_ids_list;
+    unsigned                             num_cm_id_in_quota;
```

Comment:
rename to cm_id_quota, and make it 'int' - so in case it goes negative, would be easier to detect

Reply:
IMHO cm_id_quota doesn't indicate what this param is used for...

Reply:
the work "quota" already means "quantity"